src/zcash/ZcashEngine.ts

```diff
@@ -168,6 +168,13 @@ export class ZcashEngine extends CurrencyEngine<
       })
       this.onUpdateTransactions()
     })
+    this.synchronizer.on('error', async payload => {
```

Comment:
I want to make sure the payload is not really an `unknown` type, but the library doesn't appear to have the type information that this change assume (when I checkout at least).

Reply:
You can checkout and updot the dependent PRs for types or test again once those are published and updated in the package.json in this PR